core/unit_test/TestBitManipulation.cpp

```diff
@@ -15,6 +15,7 @@
 //@HEADER
 
 #include <Kokkos_BitManipulation.hpp>
+#include <cstdint>
```

Comment:
Are you sure this include should not go in `core/src/Kokkos_NumericTraits.hpp` ?

Reply:
I don't see a reason. `Kokkos_NumericTraits.hpp` is solely for defining the traits in the `Kokkos` namespace and we shouldn't include headers that we don't need.

Reply:
:heavy_check_mark: